src/main/java/com/minecolonies/colony/jobs/JobBuilder.java

```diff
@@ -185,15 +187,28 @@ public WorkOrderBuild getWorkOrder()
     }
 
     /**
+     * Reset the needed items list.
+     */
+    private void resetNeededItems()
+    {
+        AbstractBuilding workerBuilding = this.getCitizen().getWorkBuilding();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'workerBuilding' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)